### PR TITLE
Add testimonial

### PR DESCRIPTION
**Pull request by tchret**

### CODE REVIEW

File: app/models/testimonial.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+# Indexes
+#
+#  index_testimonials_on_user_id  (user_id)
+#
+
+class Testimonial < ActiveRecord::Base
+  belongs_to :user
+  validates :en, presence: true
+  validates :fr, presence: true
```
Comment (ssaunier) [inline app/models/testimonial.rb]: ajoute un `validates :user, presence: true`

**Comment by tchret** [comment]
image

@ssaunier le render

File: app/views/api/v1/testimonials/index.jbuilder [hunk 2]
```diff
@@ -0,0 +1,8 @@
+json.array! @testimonials.each do |testimonial|
+  json.extract! testimonial, :content
+  json.user do
+    json.from_batch testimonial.user.batch.slug
```
Comment (ssaunier) [inline app/views/api/v1/testimonials/index.jbuilder]: With our schema in mind, I will go fo:

```ruby
json.user do
  json.batch do
    json.extract! testimonial.user.batch, :slug
    json.city do
      json.extract! testimonial.user.batch.city, :name
    end
  end
end
```

File: app/views/api/v1/testimonials/index.jbuilder [hunk 3]
```diff
@@ -0,0 +1,8 @@
+json.array! @testimonials.each do |testimonial|
+  json.extract! testimonial, :content
```
Comment (ssaunier) [inline app/views/api/v1/testimonials/index.jbuilder]: Add `:locale` as well.

**Comment by ssaunier** [comment]
Review done.

**Pull request merged**